teacher.rb

```diff
@@ -26,15 +18,6 @@ def teach_stuff
     response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
-
-  def receive_raise(raise)
-    @salary += raise
-  end
-
   def set_performance_rating(rating)
     response = ""
     if rating > 90
```

Comment:
Any way to refactor this method, since it's very similar to the one in `ApprenticeTeacher`?